src/main/java/com/fasterxml/jackson/databind/introspect/AnnotationIntrospectorPair.java

```diff
@@ -735,9 +735,23 @@ public JsonCreator.Mode findCreatorAnnotation(MapperConfig<?> config, Annotated
         return (mode == null) ? _secondary.findCreatorAnnotation(config, a) : mode;
     }
 
+    @Override
+    public PotentialCreator findPrimaryCreator(MapperConfig<?> config,
```

Comment:
I am not familiar with English, but is the name of this method `Primary` OK?
Personally, it seemed like a function to get a creator with a higher priority than the setting by annotation such as `JsonCreator`.
How about a name like `findImplicitCreator`?

Reply:
It's  up to debate, I'm sure -- at first I was considering canonical.
But I see the challenge wrt precedence.
Instead of implicit (which is used as sort of fallback already, which is why I hesitate with it), we could consider "default"?

`findDefaultCreator()`? (and similar naming)

Reply:
> `findDefaultCreator`

I can think of `findFallbackConstructor`, but `findDefaultCreator` would also be good.

Reply:
Ok I'll go with "default" since that to me conveys it better. But as long as descriptions define logic either would work. 
Thank you @k163377 !